app/scripts/controllers/client/ViewClientController.js

```diff
@@ -12,6 +12,85 @@
             scope.openShares = true ;
             scope.updateDefaultSavings = false;
             scope.charges = [];
+
+            // address
+            scope.addresses=[];
+            scope.view={};
+            var entityname="ADDRESS";
+            scope.addressTypeId={};
+            formdata={};
+
+
+            resourceFactory.clientTemplateResource.get(function(data)
+            {
+                scope.enableAddress=data.isAddressEnabled;
+                if(scope.enableAddress===true)
+                {
+
+                    resourceFactory.addressFieldConfiguration.get({entity:entityname},function(data){
+                        scope.view.addresstypId=data[0].is_enabled;
+                        scope.view.street=data[1].is_enabled;
+                        scope.view.address_line_1=data[2].is_enabled;
+                        scope.view.address_line_2=data[3].is_enabled;
+                        scope.view.address_line_3=data[4].is_enabled;
+                        scope.view.town_village=data[5].is_enabled;
+                        scope.view.city=data[6].is_enabled;
+                        scope.view.county_district=data[7].is_enabled;
+                        scope.view.state_province=data[8].is_enabled;
+                        scope.view.country=data[9].is_enabled;
+                        scope.view.postal_code=data[10].is_enabled;
+                        scope.view.latitue=data[11].is_enabled;
+                        scope.view.longitude=data[12].is_enabled;
+                        scope.view.isActive=data[17].is_enabled;
+                    })
+
+                    // resourceFactory.clientAddress.get({clientId:scope.client.id},function(data)routeParams.id
+                    resourceFactory.clientAddress.get({clientId:routeParams.id},function(data)
```

Comment:
Is it possible to have multiple address for a client? If yes, where you are handling it?